### ISSUE TITLE
CA Drought Map - Homepage visual

### ISSUE THREAD

**Issue by jaredwyee** [cagov/drought.ca.gov#243]
**Context:**
We want to boost the visual appeal of the drought homepage and show meaningful visualizations of current drought conditions. 

From user testing of both prototype concepts and the current drought site, we found the Drought Monitor map to be a compelling and easy to understand visual of state wide drought conditions, with color coded shading. Additionally, seeing the entire state (or the whole US on the present drought site's current conditions page), users realized the broad geographic area that is affected by drought (whole state or broader west). 

**Story:**
When I want to learn whether CA is in drought conditions, 
I want to review current and easy to understand information, 
so I can be informed and be mindful of my water usage.

**Requirements:** Refer to [Figma prototype for design spec](https://www.figma.com/file/VdQOfgIJpYD90onzb7PJSZ/Drought.ca.gov-screens?node-id=553%3A10018
)
- Add new section with gray background under quick links.
- Add 'Tracking the drought' header 
- On a weekly basis, automatically fetch [California specific Drought Monitor Map](https://droughtmonitor.unl.edu/CurrentMap/StateDroughtMonitor.aspx?CA) and D0-D4 data. Update is expected on Thursday. Consider robustness of a fetch retry. Consider whether an update is not provided on Thursday or is done after a scheduled cron set for a specific time. We should prioritize resilience so content stakeholders do not have to worry about maintenance of the map and data.
- The title (eg 'All of California is in a drought') and subtitle (eg 'This map from U.S. Drought Monitor...') should update dependent on D0-D4 % data. Refer to table here for content logic [Drought Map Title and Subtitle Content Logic](https://docs.google.com/spreadsheets/d/1nBtnHDMLez0FAcX-kg3MmHBqW8ldPSnUh9nO8PsMp9M/edit#gid=0)
- For the D0-D4 data, show as % with single decimal. We may need to round. eg 99.25 -> 99.3%. 
- As shown in the Figma link, note when the map was released in the format: 'Released Month XX, YYYY. Updates weekly.'
- Add the Drought Monitor acknowledgement. 'The U.S. Drought Monitor' should be linked to: https://droughtmonitor.unl.edu/CurrentMap/StateDroughtMonitor.aspx?CA Note the gray text color for the rest of the text.
> **The U.S. Drought Monitor** is jointly produced by the National Drought Mitigation Center at the University of Nebraska-Lincoln, the United States Department of Agriculture, and the National Oceanic and Atmospheric Administration. Map courtesy of NDMC.

![image](https://user-images.githubusercontent.com/85529382/152607286-ac1cf31d-409b-495c-ada2-f1d6522a4f4c.png)

Note: Need separate update to reconcile CA map content with Current Conditions page. Can be separate ticket.

**Comment by jaredwyee** [comment]
Wont do
